views/partials/header.hbs

```diff
@@ -0,0 +1,24 @@
+<header>
+        <div class="header_wrapper">
+            <div class="header_logo">
+                <img src="/images/logo.png" alt="Shpping Site logo">
+            </div>
+            <div class="header_nav">
+                <nav>
+                    <div class="header_nav_one">
+                        <span><a class="btn" href="/">Home</a></span>
+                        <span><a class="btn" href="/products">Products</a></span>
+                    </div>
+                    <div class="header_nav_two">
+                        <div>
+                            <span><a class="btn" href="/login">Login</a></span>
+                            <span><a class="btn" href="/register">Register</a></span>
+                        </div>
+                        <div class="header_nav_two_child">
+                            <i class="fa fa-shopping-cart" aria-hidden="true"></i>

```

Comment:
be consistent with tags for the icons, in my opinion using em will be best to go